components/director/internal/domain/package/mock/api.go

```diff
@@ -121,19 +121,17 @@ components:
           type: string
 `)
 	return &graphql.APIDefinition{
```

Comment:
I believe whole `components/director/internal/domain/package/mock` directory can be deleted